### DISCUSSION TITLE
Fox Movies Middle East

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#721] [Q&A]
**Channel Name:** Fox Movies
**Channel ID:** FoxMoviesMiddleEast.us
**Country:** Middle East
**Language:** English
**Website:** xxx
**Stream URL (optional):** xxx
**Notes (optional):** xxx

**Comment by freearhey** [comment]
According to our database this channel was closed on 1 March 2024.

https://iptv-org.github.io/channels/us/FoxMoviesMiddleEast
